---
title: Regulome
layout: regulome
---
<div class="container" style="background-color:white;font-family:'Adobe Garamond Pro';font-size:22px;padding-right:20px;padding-left:20px;"> 
<br>

<h3 style="font-family:'Adobe Garamond Pro';text-align:center;pading-left:20px;padding-right:20px;color: rgb(0,94,128);"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24360274" style="font-size:36px;">Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation</a></h3>

<div class="row" style="padding-left:40px;padding-right:20px;">
	<div class="col-md-12 text-center">
		<img src="/images/RegulomeCover.jpg" style="height:650px;">
	</div>
</div>


<h3 style="font-family:'Adobe Garamond Pro';padding-right:20px;padding-left:20px;">A key finding of the ENCODE project is that the enhancer landscape of mammalian cells undergoes marked alterations during ontogeny. However, the nature and extent of these changes are unclear. As part of the NIH Mouse Regulome Project, we here combined DNaseI hypersensitivity, ChIP-seq, and ChIA-PET technologies to map the promoter-enhancer interactomes of pluripotent ES cells and differentiated B lymphocytes. We confirm that enhancer usage varies widely across tissues. Unexpectedly, we find that this feature extends to broadly transcribed genes, including Myc and Pim1 cell-cycle regulators, which associate with an entirely different set of enhancers in ES and B cells. By means of high-resolution CpG methylomes, genome editing, and digital footprinting, we show that these enhancers recruit lineage-determining factors. Furthermore, we demonstrate that the turning on and off of enhancers during development correlates with promoter activity. We propose that organisms rely on a dynamic enhancer landscape to control basic cellular functions in a tissue-specific manner.</h3>

<br>

<h3 style="font-family:'Adobe Garamond Pro';padding-right:20px;padding-left:20px;">Raw datasets the ChIP-seq data in activated B cells (other than CTCF) and CH12 cells, DHS-seq in activated B cells, RNA-seq, whole genome methylation (other than ES cells), and ChIA-PET reported in this paper can be accessed through the Short Read Archive with the accession number <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRP029721">SRP029721</a>.</h3>

<br>

<h3 style="font-family:'Adobe Garamond Pro';padding-right:20px;padding-left:20px;">Tracks for ChIA-PET, DHS hotspots, DHS footprints, and CpG methylation for activated B cells and ES cells are available <a href="http://figshare.com/articles/Tracks/893601">here</a>.<br>Please note that the CHIA-PET files include all individual PETs, which at some loci amount to more than a hundred. For reasons of space PETs were clustered in the manuscript figures as explained in Figure S2A and the Methods section.</h3>

<br>

</div> <!-- top container -->
